app/services/idv/steps/document_capture_step.rb

```diff
@@ -1,9 +1,26 @@
 module Idv
   module Steps
     class DocumentCaptureStep < DocAuthBaseStep
-      def call; end
+      def call
+        create_document_response = doc_auth_client.create_document
 
-      def form_submit; end
+        if create_document_response.success?
+          flow_session[:instance_id] = create_document_response.instance_id
+          upload_images
+        else
+          failure(create_document_response.errors.first, create_document_response.to_h)
+        end
+      end
+
+      def form_submit
+        Idv::DocumentCaptureForm.new.submit(permit(:front_image, :front_image_data_url,
+                                                   :back_image, :back_image_data_url))
+      end
+
+      def upload_images
+        response = post_images
```

Comment:
Yeah, I think this one passes if the image uploads succeed. We need a get results request here to check the authenticity.

Reply:
yup, I missed that part - it in there now.